### PR TITLE
rootfs: add util-linux

### PR DESCRIPTION
**Pull request by bergwolf**
So that kata-agent can call related commands.

Fixes: #212

Signed-off-by: Peng Tao <bergwolf@gmail.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
/test
/cc @jcvenegas @marcov 
I wonder what the footprint impact might be...

**Comment by jodh-intel** [comment]
Technically, we only need this package if we're building an initrd since only an initrd can be used for VM templating.

The problem is the rootfs builder can't know if it's going to be used for a rootfs image or an initrd image. We could only add this package if `[ "$AGENT_INIT" = "yes" ]` but that's rather a crude heuristic.

**Comment by jodh-intel** [comment]
Yeah - space. We're only using a single binary from that package, so it does seem somewhat overkill to drag in over 100 binaries. Maybe we should remove all but `hwclock`.... or consider adding `busybox` which also provides `hwclock` (as does `toybox` ftr) - named as `hwclock` with no other symlinks maybe.

File: rootfs-builder/debian/config.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ OS_VERSION=${OS_VERSION:-9.5}
 # Set OS_NAME to the desired debian "codename"
 OS_NAME=${OS_NAME:-"stretch"}
 
+PACKAGES="util-linux"
```
Comment (marcov) [inline rootfs-builder/debian/config.sh]: You dont need to add the package here, debian is reusing the config from ubuntu.

**Comment by devimc** [comment]
wow `util-linux` contains a lot of binaries, that potentially can increase attack surface, can we sync system clock by using a Golang function or library?

**Comment by jodh-intel** [comment]
+1 if that's a possibility.

**Comment by bergwolf** [comment]
Fire enough. Let me try to copy what hwclock does with golang in the agent.

**Comment by bergwolf** [comment]
Let's pursue grpc based solution since that would work for NUME as well which is missing rtc emulation.

**Pull request closed without merging**